src/strings/index.tsx

```diff
@@ -642,6 +642,7 @@ const strings = new LocalizedStrings({
     TOTAL_SEEDS_GERMINATED_ERROR: 'Germinated amount should not be more than tested amount.',
     INVALID_VALUE: 'Invalid Value',
     RECORDING_DATE_ERROR: 'This should be later than Start Date',
+    VIABILITY_TEST_START_DATE_ERROR: 'This should be later than the date seeds were collected',
```

Comment:
This may be technically incorrect.. can it be the same date as seed collection date (we may need to check with whomever gave you this text)? 
If no, we need to update the check on line 211 to <= in viability test modal.
If yes, we need to update this string.

Reply:
I created the text but the requirement is: "viability test start dates should not be allowed to start before the date that the seeds were collected", so maybe the text is incorrect

Reply:
Updated